src/TMedia.cpp

```diff
@@ -22,6 +22,16 @@
 
 #include "TMedia.h"
 
+#include "pre_guard.h"
+#include <QDir>
+#include <QFileInfo>
```

Comment:
I think you *may* get away *without* this one if you have the `QDir` one - you might want to test it locally (set the environmental variable `WITH_UPDATER` to `NO` {if using the Qt Creator IDE you can set this in the project view - the "*Batch Edit...*" button helps with this to set several things at once}) and see...

![image](https://user-images.githubusercontent.com/6163092/70390675-16cce580-19c5-11ea-92f4-6633dcdc46d6.png)

Reply:
The PR's title is `WITH_UPDATER set to NO`, so it would seem that case was tested.

Reply:
I get that!

I was trying to point out an easy way to set the variable - rather than having to set it in a shell before running Qt Creator (or qmake) from that shell with a modified environment containing that variable...